### ISSUE TITLE
Move the g-code preview view options to drop down settings button like other panels

### ISSUE THREAD

**Issue by alexrmay91** [fluidd-core/fluidd#690]
### Is your feature request related to a problem? Please describe

![image](https://user-images.githubusercontent.com/67204669/169674475-6f3ec443-cfb9-4da0-8716-442c4f9adbe5.png)

I keep Fluidd open on my second monitor, which is in a vertical/portrait mode. This causes some issues with how Fluidd is rendered, like an extremely small g-code preview window.

### Describe the solution you'd like

![image](https://user-images.githubusercontent.com/67204669/169674558-caa8a159-bf96-43b2-acdf-3509f5092aeb.png)

If the view options were moved to a drop down settings button (like the console panel), there would be a lot more space for a larger g-code preview window.

### Describe alternatives you've considered

None.

**Comment by pedrolamas** [comment]
Fair point, makes complete sense and it is way more consistent with the rest!

I think we can take a look at this once #693 gets merged!

**Comment by alexrmay91** [comment]
You're on top of it! Much appreciated
